### PR TITLE
Tabs component

### PR DESCRIPTION
**Pull request by AlleeX**
# What does this PR do:

    Tabs component

vokoscreen-2017-10-27_11-14-30

# Where should the reviewer start:

    Diff

### CODE REVIEW

File: components/tabs/tab.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
+
+/**
+ * Tab component
+ */
+class Tab extends Component {
+  handleTabClick = (event) => {
```
Comment (froskie) [inline components/tabs/tab.js]: Could you let me know in which proposal stage this is?

Comment (AlleeX) [reply components/tabs/tab.js]: @froskie ?

Comment (AlleeX) [reply components/tabs/tab.js]: selection_164

File: components/tabs/tab.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    const { value } = this.props;
```
Comment (yurist38) [inline components/tabs/tab.js]: How about destructuring `onClick` as well as `value`?

File: components/tabs/tab.js (hunk 1 above)
Comment (spzm) [reply components/tabs/tab.js]: stage 3 as I see https://github.com/tc39/proposal-class-fields

File: tests/unit/tabs/tabs.spec.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
+import { shallow, mount } from 'enzyme';
+import React from 'react';
+import Tabs from '../../../components/tabs/tabs';
+import Tab from '../../../components/tabs/tab';
+
+describe('Radio Button', () => {
```
Comment (mAiNiNfEcTiOn) [inline tests/unit/tabs/tabs.spec.js]: :heart:

**Pull request merged**